code/modules/reagents/reagent_containers/food/drinks.dm

```diff
@@ -281,7 +281,9 @@ If you add a drink with no empty icon sprite, ensure it is flagged as NO_EMPTY_I
 
 /obj/item/reagent_containers/food/drinks/waterbottle
 	name = "bottled water"
-	desc = "Introduced to the vending machines by Skrellian request, this water comes straight from the Martian poles."
+	desc = "A fresh bottle of water from the finest plants on Silversun."
```

Comment:
wait, plants as in foliage or plants as in buildings?

Reply:
Clarified.